### DISCUSSION TITLE
workspaces question

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#4046] [Q&A]
In https://wiki.hyprland.org/Configuring/Workspace-Rules/#syntax, it says that syntax is:
`workspace=WORKSPACE,RULES`. 

However the example below is:
 `workspace=DP-1,3,rounding:false,bordersize:0`. 
How is "DP-1,3" interpreted? Thanks.

**Comment by vaxerski** [comment]
who the fuck put that shit in

`MONITOR,WORKSPACE` is parsed because of the legacy usage of `workspace=` which worked like this.

Fixed the example.
